verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -323,16 +374,21 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
         this.unlisten();
     }
 
-    deleteCase(rowData: TableRow): Promise<unknown> {
-        return new Promise((resolve, reject) => {
-            const deleteUrl = this.state.url + rowData.id;
-            this.setState({ error: '' });
-            const response = axios.delete(deleteUrl);
-            response.then(resolve).catch((e) => {
-                this.setState({ error: e.toString() });
-                reject(e);
-            });
+    async deleteCases(): Promise<void> {
+        if (this.hasSelectedRowsAcrossPages()) {
+            // TODO: Implement action for all rows
+            alert('Action not yet implemented when all rows selected');
+            return;
+        }
+        await axios.delete('/api/cases', {
```

Comment:
I think you have to try/catch otherwise if an error occurs here the UI will crash

Reply:
Ah yes, thanks. Done.